x-pack/filebeat/input/netflow/decoder/v9/lru.go

```diff
@@ -0,0 +1,172 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package v9
+
+import (
+	"bytes"
+	"container/heap"
+	"sync"
+	"time"
+
+	"github.com/elastic/beats/v7/libbeat/common/atomic"
```

Comment:
nit: any reason to not use 
https://pkg.go.dev/sync/atomic#Bool instead of libbeat wrapper?

Reply:
no particular reason, thanks for catching this